### ISSUE TITLE
Home page heading does not describe content

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4035]
## Area or Component
Static site homepage - simplereport.gov

## Quick Summary
The main heading on simplereport.gov is a heading wrapped around an image. This means the first H1 on the page reads "SimpleReport logo" when it should just read "SimpleReport"

## Accessibility Violation
Heading does not describe content: https://cdc-devtools.dequecloud.com/rule-help/en/heading-not-descriptive

## Action Requested
Change the alt text to just "SimpleReport", or make it a real heading with the logo attached instead of using the logo image as text

## Screenshots (optional)
![Screen Shot 2022-07-21 at 5 49 01 PM](https://user-images.githubusercontent.com/80282552/180338428-b4fe6d27-29fc-4d76-b456-b88fad28906e.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=856291185&range=35:35
